### PR TITLE
Sunrise FAQ

### PR DESCRIPTION
**Pull request by MichaelMacaulay**

### CODE REVIEW

File: website/pages/en/sunrise.mdx [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+The sunrise of decentralized data is an initiative spearheaded by Edge & Node, working on The Graph. The goal is to seamlessly enable subgraph developers and data consumers to upgrade to The Graph’s decentralized network.
+
+This plan draws on many previous developments from The Graph ecosystem, including an upgrade Indexer to serve queries on newly published subgraphs, and the ability to integrate new blockchain networks to The Graph.
+
+### What are the phases of the sunrise of decentralized data?
+
+**Sunray**: Enable support for hosted service chains, offer a seamless upgrade flow, offer a free plan on the network  
+**Sunbeam**: The upgrade window that subgraph developers will have to upgrade their subgraphs to The Graph Network. The start and actual length of this will be announced soon.  
+**Sunrise**: Hosted service endpoints will expire as traffic moves to The Graph Network
```
Comment (benface) [inline website/pages/en/sunrise.mdx:15]: Only one of these 3 points has periods at the end of sentences, so maybe the other 2 should as well?

File: website/pages/en/sunrise.mdx [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+Additionally, The Graph Network is significantly more robust, reliable, and cost-efficient than anything provided by a single organization or team. Hundreds of independent Indexers around the world power The Graph Network, ensuring safety, security and redundancy.
+
+That being said, if you’re still interested in running a [Graph Node](https://github.com/graphprotocol/graph-node), consider joining The Graph Network [as an Indexer](https://thegraph.com/blog/how-to-become-indexer/) to earn indexing rewards and query fees by serving data on your subgraph and others.
+
+### Should I use a centralized indexing provider?
+
+If you are building in web2, you should. If you are building in web3, the moment you use a centralized indexing provider, you are giving them control of your dapp and data. The Graph’s decentralized network offers [superior quality of service](https://thegraph.com/blog/qos-the-graph-network/), reliability with unbeatable uptime thanks to node redundancy, as well as significantly [lower costs](/network/benefits/), and you won’t be held hostage at the data layer.
```
Comment (benface) [inline website/pages/en/sunrise.mdx:35]: > If you are building in web2, you should.

😂

File: website/pages/en/sunrise.mdx [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+
+Resilience and Redundancy: Decentralized systems are inherently more robust and resilient due to their distributed nature. Data isn't stored on a single server or location. Instead, it's served by hundreds of independent Indexers around the globe. This reduces the risk of data loss or service interruptions if one node fails, leading to exceptional uptime (99.99%).
+
+Quality of Service: In addition to the impressive uptime, The Graph Network features a ~106ms median query speed (latency), and higher query success rates compared to hosted alternatives. Read more in [this blog](https://thegraph.com/blog/qos-the-graph-network/).
+
+Censorship Resistance: Centralized systems can become targets for censorship, either through regulatory pressures or network attacks. In contrast, decentralized systems, due to their dispersed architecture, are much harder to censor, ensuring continuous data availability.
+
+Transparency and Trust: Decentralized systems operate openly, enabling anyone to independently verify the data. This transparency builds trust among network participants, as they can verify the system's integrity without relying on a central authority.
```
Comment (benface) [inline website/pages/en/sunrise.mdx:47]: Should the different points listed above ("Resilience and Redundancy", "Quality of Service", etc.) be...
- Bold?
- Headings? (####)
- Preceded by bullet points?

File: website/pages/en/sunrise.mdx (hunk 1 above)
Comment (benface) [reply website/pages/en/sunrise.mdx:15]: And a comment on the content... is it normal that there are no dates at all here, not even estimates, not even for Sunray?

File: website/pages/en/sunrise.mdx [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+
+However, some subgraphs may not be eligible for indexing rewards, and as a result, may have difficulty attracting further Indexers. For example, indexing rewards may not be available for subgraphs on certain chains. Members from these blockchain communities are encouraged to integrate their chain through the [Chain Integration Process](/chain-integration-overview/).
+
+## WHat is an upgrade Indexer?
+
+### What is an upgrade Indexer? What does "upgrade Indexer" mean?
+
+It is designed to improve the experience of upgrading subgraphs from the hosted service to The Graph Network and supporting new versions of existing subgraphs that have not yet been indexed.
+
+The upgrade Indexer is aimed at bootstrapping chains that do not yet have indexing rewards on the network, as well as a fallback for new subgraph versions. The goal is to ensure that an Indexer is available to serve queries as quickly as possible after a subgraph is published.
```
Comment (benface) [inline website/pages/en/sunrise.mdx:63]: The title says "_an_ upgrade Indexer" while this says "_The_ upgrade Indexer", so I find it a bit confusing... is there just one, or are there multiple upgrade Indexers?

File: website/pages/en/sunrise.mdx [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+
+Just as you've chosen your blockchain network for its decentralized nature, security, and transparency, opting for The Graph Network is an extension of those same principles. By aligning your data infrastructure with these values, you ensure a cohesive, resilient, and trust-driven development environment.
+
+### Will my hosted service subgraph be supported by the upgrade Indexer?
+
+Yes, the upgrade Indexer will support all subgraphs published during the upgrade window.
+
+However, some subgraphs may not be eligible for indexing rewards, and as a result, may have difficulty attracting further Indexers. For example, indexing rewards may not be available for subgraphs on certain chains. Members from these blockchain communities are encouraged to integrate their chain through the [Chain Integration Process](/chain-integration-overview/).
+
+## WHat is an upgrade Indexer?
```
Comment (benface) [inline website/pages/en/sunrise.mdx:57]:
```suggestion
## What is an upgrade Indexer?
```

File: website/pages/en/sunrise.mdx [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+
+### What is an upgrade Indexer? What does "upgrade Indexer" mean?
```
Comment (benface) [inline website/pages/en/sunrise.mdx:59]: Is the repetition of "What is an upgrade Indexer?" (same as the parent heading) intentional?

File: website/pages/en/sunrise.mdx [hunk 7]
```diff
@@ -0,0 +1,117 @@
...
+
+All Indexers are welcome to sync subgraphs across chains (irrespective of indexing rewards status), including any of the same subgraphs that the upgrade Indexer will be syncing.
+
+Indexers can also become upgrade Indexers as well. Use your Indexer operator wallet and request access to all hosted service subgraph data [here](https://app.deform.cc/form/de2a6ca9-c787-4a63-a32b-14f3a5f1bc7d/).
+
+### What are the criteria for the upgrade Indexer to stop supporting a subgraph?
+
+The upgrade Indexer will serve a subgraph until it is sufficiently and successfully served with consistent queries served by at least 3 other Indexers.
+
+The upgrade Indexer will stop supporting a subgraph if it has not been queried in the last 30 days.
```
Comment (benface) [inline website/pages/en/sunrise.mdx:115]:
```suggestion
Furthermore, the upgrade Indexer will stop supporting a subgraph if it has not been queried in the last 30 days.
```

**Review by azf20 — CHANGES_REQUESTED** [review]
I think it is worth adding:
- An indication that we will continue to add to this FAQ as there is more information to share
- Where to go if you have questions

File: website/pages/en/sunrise.mdx (hunk 2 above)
Comment (azf20) [reply website/pages/en/sunrise.mdx:35]: (specifically the first sentence)

File: website/pages/en/sunrise.mdx (hunk 1 above)
Comment (MichaelMacaulay) [reply website/pages/en/sunrise.mdx:15]: Got those periods. Yes, this was intentional not to include dates.

File: website/pages/en/sunrise.mdx (hunk 2 above)
Comment (MichaelMacaulay) [reply website/pages/en/sunrise.mdx:35]: Good call - removing!

File: website/pages/en/sunrise.mdx (hunk 3 above)
Comment (MichaelMacaulay) [reply website/pages/en/sunrise.mdx:47]: Yes, good catch. Thank you! Bold + bullet points looks best locally

File: website/pages/en/sunrise.mdx (hunk 6 above)
Comment (MichaelMacaulay) [reply website/pages/en/sunrise.mdx:59]: No, this seems to be a typo - thank you for flaging!

File: website/pages/en/sunrise.mdx (hunk 4 above)
Comment (MichaelMacaulay) [reply website/pages/en/sunrise.mdx:63]: So, there is only one initially but other Indexers are encouraged to become upgrade Indexers as well. We already have 2 or 3 folks interested, with more potentially joining. However, there is no hard commitment. Hence, the hesitation around this.

This language comes directly from a google doc that _many_ people have reviewed, so I lean toward keeping the phrasing as is, and changing later if needed. Does that make sense?

**Pull request merged**